doc/source/security/encrypting-tables.rst

```diff
@@ -0,0 +1,49 @@
+.. _encrypting-tables:
+
+=========================================================
+Encrypting File-Per-Tablespace Tables
+=========================================================
+
+When you define the variable ``innodb_file_per_table`` in the my.cnf file, InnoDB uses a tablespace file for each InnoDB table and creates and stores the table data and the indexes in a single data file. In this tablespace configuration, each table is stored in an .ibd file.
```

Comment:
You do not have to define innodb_file_per_table in the my.cnf file as this is ON by default.

Reply:
Fixed.

Reply:
Still starts with "When you define the variable ``innodb_file_per_table`` in the my.cnf file" - but you don't have to define this, this is the default.